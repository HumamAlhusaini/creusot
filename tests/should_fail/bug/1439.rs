// SHORT_ERROR
extern crate creusot_contracts;
use creusot_contracts::*;

#[requires(match opt { })]
fn foo(opt: Option<i32>) {}
