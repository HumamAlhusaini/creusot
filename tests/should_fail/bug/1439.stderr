1439.rs:5:18: error[E0004]: non-exhaustive patterns: `None` and `Some(_)` not covered: patterns `None` and `Some(_)` not covered
error: aborting due to 1 previous error
